java/java.editor.base/src/org/netbeans/modules/java/editor/base/semantic/SemanticHighlighterBase.java

```diff
@@ -1103,9 +1103,14 @@ public Void visitLiteral(LiteralTree node, Void p) {
 
             TreePath pp = getCurrentPath().getParentPath();
             if (pp.getLeaf() != null &&
-                pp.getLeaf().getKind() == Kind.METHOD_INVOCATION) {
-                MethodInvocationTree inv = (MethodInvocationTree) pp.getLeaf();
-                int pos = inv.getArguments().indexOf(node);
+               (pp.getLeaf().getKind() == Kind.METHOD_INVOCATION || pp.getLeaf().getKind() == Kind.NEW_CLASS)) {
```

Comment:
Maybe, should fix the indent. Now, there are 3 spaces, but 4 or 8 spaces are better.

Reply:
Sorry, my fault. Corrected!